cli/azd/pkg/infra/azure_resource_manager.go

```diff
@@ -75,7 +76,7 @@ func (rm *AzureResourceManager) GetDeploymentResourceOperations(ctx context.Cont
 
 // GetResourceGroupsForDeployment returns the names of all the resource groups from a subscription level deployment.
 func (rm *AzureResourceManager) GetResourceGroupsForDeployment(ctx context.Context, subscriptionId string, deploymentName string) ([]string, error) {
-	deployment, err := rm.azCli.GetSubscriptionDeployment(ctx, subscriptionId, deploymentName)
+	deployment, err := azsdk.GetSubscriptionDeployment(ctx, subscriptionId, deploymentName)
```

Comment:
Felt like Wallace was taking the approach of doing the conversion of everything first and would then break it into pieces vs your approach here about cleaving things off one at a time.

I'm happy with either, but I don't want you to get in one another's way so bringing it up.

Reply:
Updated it to follow Wallace's strategy :)